src/scrapers/poalim-business/helpers.js

```diff
@@ -0,0 +1,312 @@
+              };
+              creditCardDollarsTransactions.push(transaction);
+            }
+          }
+
+          if (afterEuros) {
+            for (
+              let creditCardTransactionsIndex = 0;
+              creditCardTransactionsIndex < afterEuros.length;
+              creditCardTransactionsIndex += 1) {
+              const transaction = {
+                amount: afterEuros[creditCardTransactionsIndex].debitAmount,
+                currency: afterEuros[creditCardTransactionsIndex].debitCurrencyCode,
+                date: afterEuros[creditCardTransactionsIndex].eventDate,
+                business: (afterEuros[creditCardTransactionsIndex].clientBusinessName) ? afterEuros[creditCardTransactionsIndex].clientBusinessName.substring(
+                  afterEuros[creditCardTransactionsIndex].clientBusinessName.indexOf('~') + 1,
+                ) : '',
+                eventId: afterEuros[creditCardTransactionsIndex].originalSystemEventId,
+                reference: afterEuros[creditCardTransactionsIndex].referenceNumber,
+                debitDate: afterEuros[creditCardTransactionsIndex].debitDate,
+                originalAmount: (afterEuros[creditCardTransactionsIndex].originalAmount !=
+                      afterEuros[creditCardTransactionsIndex].debitAmount) ? afterEuros[creditCardTransactionsIndex].originalAmount : undefined,
+                originalCurrency: (afterEuros[creditCardTransactionsIndex].originalAmount !=
+                      afterEuros[creditCardTransactionsIndex].debitAmount) ? afterEuros[creditCardTransactionsIndex].eventCurrencyDescription : undefined,
+              };
+              creditCardEuroTransactions.push(transaction);
+            }
+          }
+        }
+      }
+    });
+
+    let creditShekelsBalance = 0;
+    const creditShekelsAbroadBalance = 0;
+    let creditDollarsBalance = 0;
+    let creditEurosBalance = 0;
+    if (creditCardTxnsResult) {
+      for (
+        let creditCardAccountbalancesIndex = 0;
+        creditCardAccountbalancesIndex < creditCardTxnsResult.plasticCardAccountSummeryList.length;
+        creditCardAccountbalancesIndex += 1) {
+        switch (creditCardTxnsResult.plasticCardAccountSummeryList[creditCardAccountbalancesIndex].debitCurrencyCode) {
+          case 0:
+            creditShekelsBalance =
+                creditCardTxnsResult.plasticCardAccountSummeryList[creditCardAccountbalancesIndex].debitAmount;
+            break;
+          case 19:
+            creditDollarsBalance =
+                creditCardTxnsResult.plasticCardAccountSummeryList[creditCardAccountbalancesIndex].debitAmount;
+            break;
+          case 100:
+            creditEurosBalance =
+                creditCardTxnsResult.plasticCardAccountSummeryList[creditCardAccountbalancesIndex].debitAmount;
+            break;
+
+          default:
+            break;
+        }
+      }
+    }
+
+    accounts.push({
```

Comment:
there is a lot here that is different from the format you currently are using.

Maybe we can start a discussion about that?